code/modules/gear_presets/uscm_medical.dm

```diff
@@ -159,25 +134,18 @@
 	utility_hat = list()
 	utility_gloves = list()
 	utility_shoes = list(/obj/item/clothing/shoes/laceup)
-	utility_extra = list(/obj/item/clothing/suit/storage/labcoat/researcher)
+	utility_extra = list()
 
 	service_under = list(/obj/item/clothing/under/marine/officer/researcher)
 
 /datum/equipment_preset/uscm_ship/uscm_medical/researcher/load_gear(mob/living/carbon/human/new_human)
-	var/back_item = /obj/item/storage/backpack/marine/satchel
+	var/back_item = /obj/item/storage/backpack/satchel
 	if (new_human.client && new_human.client.prefs && (new_human.client.prefs.backbag == 1))
 		back_item = /obj/item/storage/backpack/marine
 
-	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/research(new_human), WEAR_L_EAR)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/marine/officer/researcher(new_human), WEAR_BODY)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/laceup(new_human), WEAR_FEET)
 
-	if(new_human.disabilities & NEARSIGHTED)
-		new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/science/prescription(new_human), WEAR_EYES)
-	else
-		new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/science(new_human), WEAR_EYES)
```

Comment:
Why remove this? At least give them prescription glasses if they have the trait on spawn. Or implement their vendors to give them the prescription variant. Or allow buying a `/obj/item/clothing/glasses/regular` (this and `/obj/item/clothing/glasses/sunglasses/prescription` can be inserted into any `/obj/item/clothing/glasses` to give it prescription element.